webapp/src/components/legal_hold_table/legal_hold_row/legal_hold_row.tsx

```diff
@@ -136,6 +173,44 @@ const LegalHoldRow = (props: LegalHoldRowProps) => {
                         </span>
                     </a>
                 </OverlayTrigger>
+                <OverlayTrigger
```

Comment:
OverlayTrigger was recently deprecated in the webapp because it was the source of a number of memory leaks. I'm not sure if there is an alternative available anyway because we are on such an old version of the webapp but just something to be aware of

Reply:
Added a few `TODO` comments so we don't forget about that, thanks.

Reply:
Need a ticket to not forget about it.